packages/android_sdk_tools.rb

```diff
@@ -0,0 +1,28 @@
+require 'package'
+
+class Android_sdk_tools < Package
+  description 'Android SDK CLI tools'
+  homepage 'https://developer.android.com/studio'
+  version '7583922'
+  license 'Apache-2.0'
+  compatibility 'x86_64'
+  source_url 'https://dl.google.com/android/repository/commandlinetools-linux-7583922_latest.zip'
+  source_sha256 '124f2d5115eee365df6cf3228ffbca6fc3911d16f8025bebd5b1c6e2fcfa7faf'
+
+  depends_on 'jdk11'
+
+  def self.install
+    FileUtils.mkdir_p CREW_DEST_PREFIX + '/bin'
+    FileUtils.mkdir_p CREW_DEST_PREFIX + '/share/android-sdk-tools'
+    FileUtils.mv Dir['*'], CREW_DEST_PREFIX + '/share/android-sdk-tools'
+    Dir["#{CREW_DEST_PREFIX}/share/android-sdk-tools/bin/*"].each do |filename|
+      binary = File.basename(filename)
+      FileUtils.ln_s "#{CREW_PREFIX}/share/android-sdk-tools/bin/#{binary}", "#{CREW_DEST_PREFIX}/bin/#{binary}"
```

Comment:
will doing
```
Dir.chdir "#{CREW_DEST_PREFIX}/bin" do
  FileUtils.ln_s "../share/android-sdk-tools/bin/#{binary}", "#{binary}"
end
```
avoid symlinks issues?

Reply:
I'm not seeing an issue with the binaries that have the absolute symlinks.  Did you get a chance to give it a try yet?

Reply:
You are right.  I had `export CREW_SHRINK_ARCHIVE=0` in my `~/.bashrc`.  Once I removed, the symlinks broke.

Reply:
Do you think it makes sense to not run symlink in `CREW_DEST_DIR` while building or installing packages?

Reply:
I went ahead and changed to relative symlinks for the time being.

Reply:
symlinks I think fixes issues for some packages? But the relative symlinks avoids the issue.

Reply:
What exactly does it fix?  Like upx, it introduces more problems than it solves.

Reply:
That's a valid question. symlinks came in through https://github.com/skycocker/chromebrew/pull/5837

The idea was to remove absolute symlinks and make them relative, so that if CREW_PREFIX is changed, programs still work.

upx is another issue.... but it does dramatically reduce the size of packages.